staging/src/k8s.io/apimachinery/pkg/apis/meta/fuzzer/fuzzer.go

```diff
@@ -186,15 +186,15 @@ func v1FuzzerFuncs(codecs runtimeserializer.CodecFactory) []interface{} {
 			j.ResourceVersion = strconv.FormatUint(c.RandUint64(), 10)
 			j.UID = types.UID(c.RandString())
 
-			var sec, nsec int64
+			var sec, nsec uint32
```

Comment:
I was thinking `int32`, but `uint32` is fine. Could you add a comment "Fuzzing sec and nsec in a smaller range (uint32 instead of int64), so that the result Unix time is a valid date and can be parsed into RFC3339 format".

Reply:
Done, thanks — good catch!